GunShipAlpha/src/GameState.java

```diff
@@ -21,8 +21,8 @@ public void update(Player player) {
             if (!isBehemothBlocked(enemy)) {
                 enemy.moveTowardTarget();
             }
-            if (enemy instanceof Tank) {
-                Tank tank = (Tank) enemy;
+            if (enemy instanceof Mine) {
```

Comment:
see earlier comment about instanceof. There is probably a superclass method hiding here.